vue.js-case-study.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Vue.js Case Study
+author: Lars Halvor Vikse Kallåk <lhvk>
+tags: javascript, framework, vue
+---
+
+## Introduction
+ 
+ ![Image of Vue logo](https://res.cloudinary.com/practicaldev/image/fetch/s--q6AIFazN--/c_imagga_scale,f_auto,fl_progressive,h_420,q_auto,w_1000/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/ll22z8mea106b4vjdjy7.jpeg)
+
+I've chosen Vue as my case study because frontend is what I study. I've never used Vue and I have zero experience with it, so I reckon this would be pretty interesting to dive deeper into.
+
+## Brief History
+
+Vue was created by Evan You, a Google software engineer, in 2013 and had its official release in 2014. The name derives from the French word vue, meaning view. Evan wanted to create a framework that was lightweight and included the best parts from Angular, which he had been using at Google.
+
+#### Here is a timeline highlighting the major events:
+ 
+- **2013:** Version 0.6 In December Evan You created the first version of Vue
+- **2014:** Version 0.8 First public release
+- **2015:** Version 1.0 Released
+- **2016:** Version 2.0 Released
+- **2020:** Version 3.0 Released
+
+## Main Features
+ 
+Vue is a Javascript framework used to build frontend UI. It is simple and has a small learning curve, good library support and is efficient.
+
+| Feature | Description |
+| :--- | :--- |
+| Virtual DOM | Virtual representation of the DOM tree that makes the page faster |
+| Data Binding | Manipulates/assign values to HTML attributes, change style or assign classes etc. |
+| Components | Create custom reusable HTML elements |
+| Event Handling | Event listeners |
+| Animation/Transition | Built-in wrappers and/or third party libraries for transition effects |
+| Computed Properties | Listens to changes made to UI elements and performs necessary calculations without additional coding |
+| Templates | Compiles templates into virtual DOM render functions |
+| Directives | Buil-in directives such as: v-if, v-else, v-show, v-on, v-bind, and v-model |
+| Watchers | Applied to changing data such as form inputs. Data handling is taken care of by a watcher |
+| Routing | Vue-router helps with navigation between pages |
+| Lightweight | Light and fast performing |
+
+## Market Comparison
+
+ ### Vue vs React
+ 
+#### Virtual DOM
+ - Both use virtual DOM which makes it faster.
```

Comment:
```suggestion
 - Both use virtual DOM which makes it faster.
```

It would be nice to know why virtual DOM is considered good for performance, especially when other frameworks like Svelte intentionally do not use this.